telegram/ext/filters.py

```diff
@@ -272,6 +272,10 @@ def filter(self, message):
             ...
             MessageHandler(Filters.text(buttons), callback_method)
 
+    Note:
+        Dice messages don't have text. If you want to treat them as text messages anyway, use
+        ``Filters.text | Filters.dice``.
```

Comment:
Is that accurate? How does that work?

Reply:
Yeah, that's rather a remnant of my previous attempt. Rephrased.

Reply:
If you want to __filter__ either ...